etc/dbus-serialbattery/config.default.ini

```diff
@@ -228,6 +228,28 @@ TIME_TO_SOC_RECALCULATE_EVERY = 60
 ; These will be as negative time. Disabling this improves performance slightly
 TIME_TO_SOC_INC_FROM = False
 
+; --------- SOC calculation ---------
+; Description: Calculate the SOC in the driver. Do not use the SOC reported by the BMS
+; SOC_CALCULATION = True: Calc SOC in the driver, do not use SOC reported from BMS
+;      - The SOC is calculated by integration of the current reported by the BMS.
+;      - If SOC_CALC_CURRENT_CORRECTION = True then a correction applies to the current defined by
+;        the map (SOC_CALC_CURRENT_MEASURED,SOC_CALC_CURRENT_REAL).
+;      - If SOC_CALC_CURRENT_CORRECTION = False then the current reported by the bms is integrated as it is.
+;      - The SOC is set to 100% if the following conditions apply for at least SOC_RESET_TIME seconds:
+;          * current is lower than SOC_RESET_CURRENT Amps
+;          * Sum of Cell voltages >= MAX_CELL_VOLTAGE * Cell Count - VOLTAGE_DROP
+;      - When the driver starts, the SOC starts at SOC_CALC_INIT_VALUE if SOC_CALC_RESET_VALUE_ON_RESTART is set.
+;        If SOC_CALC_RESET_VALUE_ON_RESTART=False then it is tried to restore the Soc from the dbus. If it is not found,
+;        the Soc is resetted to SOC_CALC_INIT_VALUE nevertheless.
+; SOC_CALCULATION = False: Use SOC reported from BMS (none of the other parameters apply)
+SOC_CALCULATION = False
+SOC_RESET_CURRENT = 7
+SOC_RESET_TIME = 60
+SOC_CALC_CURRENT_CORRECTION = False
+SOC_CALC_CURRENT_MEASURED = -300, 300
+SOC_CALC_CURRENT_REAL = -300, 300
+SOC_CALC_RESET_VALUE_ON_RESTART = False
+SOC_CALC_INIT_VALUE = 25
```

Comment:
Please move this below the section `; -- CVL reset based on SoC option (step mode & linear mode)` so we have similar settings in the same place.

In my eyes this are too much options here. We need to keep it as simple as possible

I would use the existing `MAX_VOLTAGE_TIME_SEC` instead of `SOC_RESET_TIME`.

`SOC_CALC_CURRENT_CORRECTION = False` could be removed. Just check, if `SOC_CALC_CURRENT_MEASURED` and `SOC_CALC_CURRENT_REAL` are the same on driver start.

I would also remove `SOC_CALC_RESET_VALUE_ON_RESTART` and `SOC_CALC_INIT_VALUE`. If no SOC is found in dbus, then use the BMS SOC to start. I think it would make no sense to start from a specific value on every driver restart or device reboot.

What do you think about this?

Reply:
HI @mr-manuel, thank you very much for your remark.
I added the Option SOC_CALC_RESET_VALUE_ON_RESTART and SOC_CALC_INIT_VALUE to be able to synchronize with the smartshunt on start. But I am fine with your suggestion too. I agree that it is important to keep it simple.
The other topics are totally fine for me. I will change tomorrow evening.
Thank you!

Reply:
If it ok for you I would like to keep `SOC_RESET_TIME`, because it is typically much smaller (about 60s) than `MAX_VOLTAGE_TIME_SEC`.
I removed 
`SOC_CALC_CURRENT_CORRECTION `
`SOC_CALC_RESET_VALUE_ON_RESTART`
`SOC_CALC_INIT_VALUE`

And I moved the paragraph as suggested.